web/src/models/AssertionResults.model.ts

```diff
@@ -11,6 +11,7 @@ const AssertionResults = ({allPassed = false, results = []}: TRawAssertionResult
       id: uniqueId(),
       selector,
       spanIds: AssertionService.getSpanIds(resultList),
+      originalSelector: selector,
```

Comment:
creating the field on AsserionResult

Reply:
Do we want this in the model? I see we also have a `assertionResultsToDefinitionList` in `web/src/redux/slices/TestDefinition.slice.ts`

Reply:
I think it is good here for the moment, but I get your point